### PR TITLE
[LLVM] Change error messages to start with lower case

### PR DESCRIPTION
**Pull request by jurahul**
Change LLVM Asm and TableGen Lexer/Parser error messages to begin with lower case.

### CODE REVIEW

File: llvm/lib/AsmParser/LLLexer.cpp [hunk 1]
```diff
@@ -60,8 +60,8 @@ uint64_t LLLexer::atoull(const char *Buffer, const char *End) {
     uint64_t OldRes = Result;
     Result *= 10;
     Result += *Buffer-'0';
-    if (Result < OldRes) {  // Uh, oh, overflow detected!!!
-      LexError("constant bigger than 64 bits detected!");
+    if (Result < OldRes) { // overflow detected.
```
Comment (arsenm) [inline llvm/lib/AsmParser/LLLexer.cpp:63]: Not an error message

**Review by arsenm — COMMENTED** [review]
https://llvm.org/docs/CodingStandards.html#error-and-warning-messages

File: llvm/lib/TableGen/TGLexer.cpp [hunk 2]
```diff
@@ -156,7 +156,7 @@ int TGLexer::getNextChar() {
       return EOF;
     }
     PrintError(getLoc(),
-               "NUL character is invalid in source; treated as space");
+               "nul character is invalid in source; treated as space");
```
Comment (arsenm) [inline llvm/lib/TableGen/TGLexer.cpp:159]: Seems like a special case to leave as is

File: llvm/lib/AsmParser/LLLexer.cpp (hunk 1 above)
Comment (jurahul) [reply llvm/lib/AsmParser/LLLexer.cpp:63]: Agreed, but this is just changing the comment to be "less excited" :)

File: llvm/lib/TableGen/TGLexer.cpp (hunk 2 above)
Comment (jurahul) [reply llvm/lib/TableGen/TGLexer.cpp]: Done, and also changes a similar error in Asm lexer to be consistent in wording with this one (null bytes -> NUL character)

**Review by mshockwave — APPROVED** [review]
LGTM

**Comment by jurahul** [comment]
@arsenm any further comments?

**Comment by jurahul** [comment]
> LGTM

Thanks. Will give @arsenm a final chance to look before merging.

**Review by arsenm — APPROVED** [review]

**Pull request merged**